tests/suites/test_suite_psa_crypto.data

```diff
@@ -1210,6 +1226,10 @@ PSA symmetric encrypt/decrypt: AES-CTR
 depends_on:MBEDTLS_AES_C:MBEDTLS_CIPHER_MODE_CTR
 cipher_verify_output:PSA_ALG_CTR:PSA_KEY_TYPE_AES:"2b7e151628aed2a6abf7158809cf4f3c":"6bc1bee22e409f96e93d7e117393172a"
 
+PSA symmetric encryption multipart: AES-ECB, 16+16 bytes
+depends_on:MBEDTLS_AES_C
+cipher_encrypt_multipart:PSA_ALG_ECB_NO_PADDING:PSA_KEY_TYPE_AES:"2b7e151628aed2a6abf7158809cf4f3c":"":"6bc1bee22e409f96e93d7e117393172a5434f378a597bcef1389318c7fc865ef":16:16:16:"3ad77bb40d7a3660a89ecaf32466ef9755ed5e9e066820fa52c729886d18854c"
+
```

Comment:
There weren't any negative test cases being done with _multipart tests. Is it really up to me to introduce that here?

Reply:
A prior test gap is not a justification for expanding the test gap in general. But ok, let's [defer negative testing for multipart operations](https://github.com/ARMmbed/mbedtls/issues/3607) since it's a preexisting lack of test code. Do please add test with input that isn't split on a block boundary, which was already done for CBC.

Reply:
Sure (push incoming for the non-block-multiple operations).